README-VOUCHERIFY.md

```diff
@@ -0,0 +1,166 @@
+# Emporix B2B Showcase
+
```

Comment:
Table of contents + numeration + reasonable structure, e.g.
- introduction
  - Contentful role in this integration 
  - Qualifications
- how to prepare the environment
  - prepare Emporix, Voucherify and Contentful services 
    - migrations
    - how to get API keys
  - how to prepare showcase application
    - requirements (node, npm, ?)
    - installation process
       - download repo
       - install dependencies
       - configure .env
       -  run
    - commands? (yarn start / yarn install)
- how it works
  - Applying vouchers and promotion tiers 
  - Bundles
  - Integration from a technical perspective